fastlane/Fastfile

```diff
@@ -312,7 +312,7 @@ REPOSITORY_NAME="WordPress-Android"
 
     # Create the file names
     version=android_get_release_version()
-    build_bundle(version: version, flavor:"vanilla", buildType: "release")
+    build_bundle(version: version, app: "WordPress", flavor:"vanilla", buildType: "release")
```

Comment:
Any reason why this is titlecased here while it's lowecase everywhere else?

I guess it doesn't really matter since you call `downcase` on line 632 anyway, but would still be nice to be consistent everywhere 🙂

Reply:
Ha yeah you're right!  it doesn't matter because of the down case but I'll update this to be consistent 🙂